build.gradle

```diff
@@ -71,8 +70,9 @@ allprojects {
     group = GROUP
 
     repositories {
-        jcenter()
         google()
+        mavenCentral()
+        gradlePluginPortal()
```

Comment:
I'm a bit against this change, mostly because we're _polluting_ the runtime/compile dependencies with a repository that should not offer such dependencies (but only build plugins).

The reason why it worked is most likely because Gradle Portal is mirroring other repos.
I think we can add a:

```
// TODO Remove me once trove4j migrates away from JCenter.
jcenter {
    content {
        includeModule("org.jetbrains.trove4j", "trove4j")
    }
}
```

To restrict usage of `jcenter()` to only `trove4j`

Reply:
I agree. I don't think there is a need to rush this.